.github/workflows/ci.yaml

```diff
@@ -87,3 +87,39 @@ jobs:
           sudo snap start charmed-kafka.cruise-control
           sleep 20
           curl -s http://localhost:9090/kafkacruisecontrol/state | grep "state: RUNNING"
+
+  test-with-kraft:
+    name: Test Snap with Kraft
+    runs-on: ubuntu-latest
+    needs:
+      - build
+    steps:
+      - name: Download snap file
+        uses: actions/download-artifact@v3
+        with:
+          name: charmed-kafka_snap_amd64
+          path: .
+
+      - name: Install snap file
+        run: |
+          sudo snap install charmed-kafka_${{ needs.build.outputs.version }}_amd64.snap --dangerous
+
+      - name: Set default kraft config
+        run: |
+          sudo cp /snap/charmed-kafka/current/opt/kafka/config/kraft/server.properties /var/snap/charmed-kafka/current/etc/kafka
+
+          sudo sed -i '/log.dirs=/c\log.dirs=/var/snap/charmed-kafka/common/var/log/kafka' /var/snap/charmed-kafka/current/etc/kafka/server.properties
+          uuid=$(sudo charmed-kafka.storage random-uuid)
+          sudo charmed-kafka.storage format --cluster-id $uuid -c /var/snap/charmed-kafka/current/etc/kafka/server.properties
+
+      - name: Start snap services
+        run: |
+          sudo snap start charmed-kafka.daemon
+          sleep 5
+
+      - name: Check topic creation
+        run: |
+          topic_creation=$(charmed-kafka.topics --create --topic test --bootstrap-server localhost:9092)
```

Comment:
**question** I would have thought that we need sudo here too

Reply:
Apparently not. I copied the previous test with zookeeper, where is not needed. Didn't investigate on why though.